README.md

```diff
@@ -156,7 +156,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 
 - **To find out about how to fix a Git Conflict, go to [GitHub Docs - Resolve Merge Conflict](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)**
 
-- You can also ask for help in our [Discord server](http://discord.eddiehub.org) or submit an issue in the [Support repository](https://github.com/EddieHubCommunity/support).
+- You can also ask for help in our [Discord server](http://discord.eddiehub.org) or submit an [issue](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) in the Support repository.
```

Comment:
Not sure if the invite me to the GitHub org is the right issue here. Due the text says if there are any issues during the process you can ask in discord or open a new issue. In my opinion we should rephrase the sentence that it says joining our community.

Reply:
can you edit it as you suggest?